src/main/java/frc/robot/commands/feeder/StartFeeder.java

```diff
@@ -33,10 +33,10 @@ public void end(boolean interrupted) {
 
     @Override
     public boolean isFinished() {
-        if (feeder.pieceSeen(true)) {
-            return true;
-        }
-        else if (Timer.getFPGATimestamp() - startTime > Constants.START_FEEDER_TIMEOUT) {
+//        if (feeder.pieceSeen(true)) {
```

Comment:
I think this will break some other sequences that use it (in auto). If the other command replaces it, we should adjust all the sequences.

Reply:
pushed a fix for sequences (pathplanner fixes are in another branch)